docs/index.md

```diff
@@ -57,6 +57,10 @@
 
 Here are some [recommended resources](Resources.md) to browse
 
+## Tools
+
+Here is documentation on the [GIS to GEXF](Tools/GIStoGEXF.md) tool
```

Comment:
Maybe write what the purpose of the script within the 911 simulation project is. That way people won't be puzzled by it.